tests/multiply_matrix_x2_typed.c

```diff
@@ -85,7 +85,8 @@ int TYPED_FUNC(
     const double alpha = 1.0;
     const double beta = 0.0;
     const double threshold = 0.0;
-    const double trace;
+    REAL_T* trace;
```

Comment:
@cnegre The indentation here is not correct. Have a look at the Travis-CI error message and the diff towards the end. But basically you need to move the `*` so that it reads `REAL_T *trace;`.